DISC-NU/books-and-breakfast-app
# transportation/status screen improvements & image population

### Description

Please explain the changes you made here.
Removed signup screen file, changed rideshare filename, changed date to day of week, changed wording to be more general and casual, changed wording in profile screen to "Your Assigned Volunteering Site". Added images to mission screen and transportation screen

### QA

Please explain in detailed steps how the reviewer can test your changes and what the expected behaivors are (visual would help!). Consider different test cases/scenarios.

I can't run Android Studio currently so I haven't tested it. Let me know if the formatting is okay and if I should change any wording. Also still hard coded (don't think firebase functions are written yet)

### Packages

Please describe any additional packages you added to package.json.

### Checklist
